vit-servicing-station-lib/src/db/models/snapshot.rs

```diff
@@ -0,0 +1,165 @@
+use crate::db::{
+    schema::{contributors, snapshots, voters},
+    Db,
+};
+use diesel::{ExpressionMethods, Insertable, Queryable};
+use jormungandr_lib::crypto::account::Identifier;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Snapshot {
+    /// Tag - a unique identifier of the current snapshot
+    pub tag: String,
+    /// Timestamp for the latest update of the current snapshot
+    #[serde(alias = "lastUpdated")]
+    #[serde(deserialize_with = "crate::utils::serde::deserialize_unix_timestamp_from_rfc3339")]
+    #[serde(serialize_with = "crate::utils::serde::serialize_unix_timestamp_as_rfc3339")]
+    pub last_updated: i64,
+}
+
+impl Queryable<snapshots::SqlType, Db> for Snapshot {
+    type Row = (
+        // 0 -> tag
+        String,
+        // 1 -> last_updated
+        i64,
+    );
+
+    fn build(row: Self::Row) -> Self {
+        Self {
+            tag: row.0,
+            last_updated: row.1,
+        }
+    }
+}
+
+impl Insertable<snapshots::table> for Snapshot {
+    type Values = (
+        diesel::dsl::Eq<snapshots::tag, String>,
+        diesel::dsl::Eq<snapshots::last_updated, i64>,
+    );
+
+    fn values(self) -> Self::Values {
+        (
+            snapshots::tag.eq(self.tag),
+            snapshots::last_updated.eq(self.last_updated),
+        )
+    }
+}
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Voter {
+    #[serde(alias = "votingKey")]
```

Comment:
We could put `#[serde(rename_all = "camelCase")]` on the struct and remove these aliases?

Reply:
fixed